cmd/wigStats/testdata/test.wig

```diff
@@ -0,0 +1,11 @@
+fixedStep chrom=chr1 start=1 step=1
+1
+-10
```

Comment:
I saw in the usage that the wig needs to have non-negative integers. It looks like this test wig would break that rule?

Reply:
Seth said the negative value is the missing data value, and has updated Usage to reflect this